### DISCUSSION TITLE
Discord webhook support

### DISCUSSION THREAD

**Discussion by Heroic-Brine** [mikulash/Octoprint-detector2#9] [Ideas]
Adding support for a discord web hook would allow much faster delivery and more or less no limit (at least for a few printers)
Would also be relatively simple to implement from what i have heard from other programmers.

It should be able to post a picture as well as how sure and so on.

Just an idea but would be _very_ helpful

**Comment by mikulash** [comment]
sounds good, will look into that, thanks for the idea

**Comment by fokuleh** [comment]
Can you implement vía telegram notification?
